YesterdayApi/Utilities/Exceptions/ExceptionHandlingMiddleware.cs

```diff
@@ -0,0 +1,51 @@
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+
+namespace YesterdayApi.Utilities.Exceptions
+{
+    public class ExceptionHandlingMiddleware
+    {
+        private readonly RequestDelegate _next;
+        public ExceptionHandlingMiddleware(RequestDelegate next)
+        {
+            _next = next;
```

Comment:
> I think private field "_next" should begin with an underscore

https://github.com/dotnet/corefx/blob/368fdfd86ee3a3bf1bca2a6c339ee590f3d6505d/Documentation/coding-guidelines/coding-style.md

That's fine. Please turn off this warning.